components/match2/wikis/arenafps/match_summary.lua

```diff
@@ -47,9 +47,7 @@ function CustomMatchSummary.createBody(match)
 
 	-- Iterate each map
 	for _, game in ipairs(match.games) do
```

Comment:
array.forEach?

Reply:
Nah, not now see #4948 which is stacked on this 
https://github.com/Liquipedia/Lua-Modules/pull/4948/files#diff-a958b151989f86e16a78fe321c13425cc8efc5141d581fdfd75d2bff0274ad54